modules/maps/kivent_maps/map_utils.py

```diff
@@ -361,8 +359,8 @@ def _load_tile_map(layers, width, tile_properties):
             tile_layer_count += 1
             tile_zindex.append(i)
         elif type(layer) == ObjectGroup:
-            color = get_color_from_hex(layer.color)
-            color = (color[0]*255, color[1]*255, color[2]*255, 255)
+            c = layer.color
+            color = (c._Color__r, c._Color__g, c._Color__b, 255)
```

Comment:
@saqib1707 I think we can safely set the default to full black for now. I think the case spinningD20 is mentioning is when the layer has only images, and no polygons.Technically layer color shouldn't be used further from that point in that case, so a dummy value should be fine.